src/status_im/contexts/profile/edit/ens/view.cljs

```diff
@@ -0,0 +1,35 @@
+(ns status-im.contexts.profile.edit.ens.view
+  (:require
+    [quo.core :as quo]
+    [react-native.core :as rn]
+    [react-native.safe-area :as safe-area]
+    [status-im.contexts.profile.edit.ens.style :as style]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn view
+  []
+  (let [insets    (safe-area/get-insets)
+        user-name (rf/sub [:get-screen-params :edit-ens])]
+    [quo/overlay
+     {:type            :shell
+      :container-style (style/page-wrapper insets)}
+     [quo/page-nav
+      {:key        :header
```

Comment:
See my comment about fixing `quo/overlay`. You'll be able to remove all fixed `:key` props in children of `quo/overlay`.